pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -210,8 +262,13 @@ async def _create_or_update_pull_request(self, nvrs: List[str]):
               help="The group of components on which to operate. e.g. openshift-4.9")
 @click.option("--assembly", metavar="ASSEMBLY_NAME", required=True,
               help="The name of an assembly to rebase & build for. e.g. 4.9.1")
+@click.option("--payload", "payloads", metavar="PULLSPEC", multiple=True,
```

Comment:
Maybe I should've asked this earlier, but aren't `--payload` and `--no-rebase` mutually exclusive?  If that's the case, we could implement them with a single flag?

Reply:
They are not mutually exclusive. When building microshift for named assemblies, we don't specify either flags. `--payload` is just for STREAM and --no-rebase is just for testing purpose.

Reply:
:+1: